### DISCUSSION TITLE
Real-time Data Sync

### DISCUSSION THREAD

**Discussion by himadreekuntal** [mehedijaman/laravel-zkteco#1] [Q&A] [answered]
How Real-time Data Sync possible using the package

**Comment by mehedijaman** [comment, marked as answer]
Hello Himadree Barua, 
Right now this package is not capable of real time data sync. You can use this package to manually push/pull from/to device. 

For real time data sync you to use BioTime software by ZKTeco. Also you need device with automatic PUSH feature.

Thank you.

**Comment by himadreekuntal** [comment]
Hello Mehedi

Is there any way to do this? Can you share your idea regarding this?  

Thank you

**Reply by mehedijaman** [reply]
https://www.zkteco.com/en/ZKBioTime/ZKBioTime

See the demo here.  You will find an idea.
